src/System.Text.Json/src/Configurations.props

```diff
@@ -2,7 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp;
+      netcoreapp-Unix;
```

Comment:
@jkotas, I had to add OS-specific configurations since I need to reference CoreLib and use S.R.CS.Unsafe for the IndexOfAny helper. Is that fine?

From previous PR: https://github.com/dotnet/corefx/pull/33203#discussion_r230272227

Reply:
You should not be referencing CoreLib directly from this package. You should get `System.Runtime.CompilerServices.Unsafe` via  regular package reference.

Reply:
We can't reference System.Runtime.CompilerServices.Unsafe because it is an OOB package and System.Text.Json is going to be inbox.